lib/widgets/PaddedButton.dart

```diff
@@ -51,12 +70,14 @@ class PaddedButton extends StatelessWidget {
       this.autofocus});
 
   final EdgeInsets padding;
+  final bool sizeCover;
   final String text;
   final bool isLoading;
   final bool enabled;
   final Color? color;
   final ButtonType type;
   final Widget icon;
```

Comment:
The primary button text should be bold
![Screenshot 2024-10-07 at 16 19 18](https://github.com/user-attachments/assets/87e979e0-e91b-47d8-b8a2-c139a66ee97a)